docs/user/introduction.asciidoc

```diff
@@ -1,12 +1,160 @@
-code, no additional infrastructure required.
+* **{kib} is an open-source analytics and visualization platform.**
+Use {kib} to explore your {es} data, and then build beautiful visualizations and dashboards.
+
+* **{kib} is a UI for managing {es} clusters.**
+Manage your security settings, assign user roles, monitor the Stack,
+ingest and roll up your data...the list goes on.
+
+* **{kib} is a centralized hub for our solutions.** &mdash; From log analytics to
+document discovery to SIEM, {kib} is the portal for accessing these and other capabilities.
+
+[role="screenshot"]
+image::images/intro-kibana.png[]
+
+[float]
+[[where-to-start-kibana]]
+=== Where do I start?
+
+There is no faster way to try out {kib} than with our hosted {es} Service.
+https://www.elastic.co/cloud/elasticsearch-service/signup[Sign up for a free trial]
+and start exploring data in minutes.
+
+You can also install {kib} on your own &mdash; no code, no additional
+infrastructure required. If you don’t have {es}, you’ll need that too.
+Think of Elasticsearch as the engine that stores and processes the data, with {kib} sitting on top.
+
+[float]
+[[get-data-into-kibana]]
+=== Try {kib} with our data sets and tutorials
+
+Our <<add-sample-data, sample data sets>> enable you to take {kib} for a test ride without having
+to go through the process of loading data yourself. Or, try our <<getting-started, built-in tutorials>>,
+which can help you quickly set up data flows between your app and the Elastic Stack.
+
+[role="screenshot"]
+image::images/intro-data-tutorial.png[]
+
+With both scenarios, you get data, index patterns, visualizations, and dashboards.
+<<index-patterns, Index patterns>> are key to Kibana &mdash; they identify which
+{es} indices contain the data that you want to work with.
+
+If a tutorial doesn’t exist for your data, go to the
+{beats-ref}/beats-reference.html[Beats overview] to learn about other data shippers
+in the {beats} family.
+
+[float]
+[[explore-and-query]]
+=== Explore & query
+
+Ready to dive into your data? With <<discover, Discover>>, you can explore your data and
+search for hidden insights and relationships. Ask your questions, and then
+narrow the results to just the data you want.
+
+[role="screenshot"]
+image::images/intro-discover.png[]
+
+[float]
+[[visualize-and-analyze]]
+=== Visualize & analyze
+
+A visualization is worth a thousand log lines, and {kib} provides
+many options for showcasing your data. Use <<visualize, Visualize>> to create
+classic line charts, bar graphs, pie charts, histograms, and tables
+(just to name a few).
+
+Once your visualizations are just the way you want,
+use <<dashboard, Dashboard>> to collect them in one place. A dashboard provides
+insights into your data from multiple perspectives, and you can watch the data
+change in real time.
+
+[role="screenshot"]
+image::images/intro-dashboard.png[]
+
+[float]
+[[more-ways-to-visualize]]
+=== Even more ways to visualize
```

Comment:
I would remove this title. Line 88 does a good job of introducing this content, and these apps do more than visualize the data, they help with analysis too.